camera.go

```diff
@@ -141,6 +154,10 @@ func (cam *cameraSystem) Z() float32 {
 	return cam.z
 }
 
+func (cam *cameraSystem) Angle() float32 {
```

Comment:
I think we should try to document all exported functions when we can. Eventually there will be a PR addressing this but with new code we should try and document them as we go.